model/Software/Vocabularies/SBOMDocumentType.md

```diff
@@ -0,0 +1,25 @@
+SPDX-License-Identifier: Community-Spec-1.0
+
+# SBOMDocumentType
```

Comment:
Recommend removing the word document because "bill of materials" implies document.